src/App.js

```diff
@@ -111,7 +111,7 @@ export class App extends React.Component<AppProps, AppState> {
     interpreter: Interpreter;
     audioManager: AudioManager;
     focusTrapManager: FocusTrapManager;
-    startingCharacterState: CharacterState;
+    startingCharacterStates: any;
```

Comment:
It's trivial to make a type for all the supported character states, and I think we should unless we have a reason not to.